packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/datagrid-plus.vue

```diff
@@ -149,7 +149,7 @@ export default {
                 newParams = `${newParams}&${this.filters[i].column}${condition}=${this.filters[i].val}`;
             }
 
-            this.url = `${this.src}?v=1${newParams}`;
+            this.url = this.extraFilters?`${this.src}?v=1${newParams}&locale=${this.extraFilters.current.locale}`:`${this.src}?v=1${newParams}`;
```

Comment:
~~~js
this.extraFilters
    ? `${this.src}?v=1${newParams}&locale=${this.extraFilters.current.locale}`
    : `${this.src}?v=1${newParams}`;
~~~